src/main/java/org/kafkahq/controllers/TopicController.java

```diff
@@ -51,6 +52,12 @@
     private String defaultView;
     @Value("${kafkahq.topic.page-size:25}")
     private Integer pageSize;
+    @Value("${kafkahq.topic.replication:1}")
```

Comment:
Remove default value since they are in application.conf (to avoid false duplicate)

Reply:
I've removed them from there, but hasn't `page-size` also a default of 25 from the application file ?